lms/djangoapps/api_manager/management/commands/tests/test_migrate_courseids.py

```diff
@@ -5,22 +5,27 @@
 from datetime import datetime
 import uuid
 
+from django.conf import settings
 from django.contrib.auth.models import Group, User
+from django.test.utils import override_settings
 
 from api_manager import models as api_models
 from progress.models import CourseModuleCompletion
 from api_manager.management.commands import migrate_courseids
-from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase, mixed_store_config
 from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
 
+MODULESTORE_CONFIG = mixed_store_config(settings.COMMON_TEST_DATA_ROOT, {}, include_xml=False)
+from django.db import connection
```

Comment:
This import appears to be unused. (BTW, why isn't pylint catching it for you during the quality check?)